Firestore/CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
   that reduces the number of billed document reads when documents were deleted
   on the server while the client was not actively listening to the query
   (e.g. while the client was offline). (#INSERT_PR_NUMBER_HERE)
+- [added] Developers using Firestore on **visionOS** must use a source
+  Firestore distribution rather than the default binary distribution. To do
+  this, quit Xcode and open the desired project from the command line
+  with the `FIREBASE_SOURCE_FIRESTORE` environment variable:
+  ```
+  open --env FIREBASE_SOURCE_FIRESTORE /path/to/project.xcodeproj
+  ```
+  To go back to using the binary distribution of Firestore, quit Xcode and
+  open Xcode like normal, without the environment variable. (#11492)
```

Comment:
Hi @morganchen12, WDYT of this?

Reply:
It's not an ideal UX, but we can improve it as visionOS adoption increases. For the time being I think it's fine.